services/web/server/tests/unit/with_postgres/config.yaml

```diff
@@ -57,3 +57,8 @@ projects:
 session:
   # python -c "from cryptography.fernet import Fernet; print(Fernet.generate_key())"
   secret_key: 'tjwiMSLe0Xd9dwMlAVQT9pYY9JEnr7rcH05fkUcukVs='
+activity:
+  enabled: True
+  prometheus_host: http://prometheus
```

Comment:
This service is NOT available in this test setup. Please set ``enabled: False``  and  there is not need to add the remaining parameters if you do not want